pilot/pkg/model/config.go

```diff
@@ -837,6 +851,27 @@ func (store *istioConfigStore) Gateways(workloadLabels LabelsCollection) []Confi
 	return out
 }
 
+func (store *istioConfigStore) EnvoyFilter(workloadLabels LabelsCollection) *Config {
```

Comment:
A workload might match several `EnvoyFilter`s and the plug-in needs to consider them all, not just the first match we find.

Reply:
Yes, but one step at a time. I am just putting the basic framework in place to match basic needs as of now. 
The semantics of precedence, overrides, etc. all need to be considered and spec'd out before we allow such stuff.

Reply:
I think that limitation should be documented, otherwise it may slip through.